services/core-web/src/styles/generic/layout.scss

```diff
@@ -115,9 +115,6 @@ html {
     height: 50%;
   }
 
-  & div {
```

Comment:
This feels like it could have potential side-effects.  I've seen `inline-flex` used fairly frequently.  If we keep it, maybe give Jason a heads up to look out for odd spacing?

Reply:
I think I'll change it up a bit from what I have. I _believe_ the intention with this one is to space out the menu items in the top nav (selector compiles to `.inline-flex div`, the menu is the only change I notice).

![image](https://github.com/bcgov/mds/assets/102187683/cebc970d-cd57-492f-936a-b75b315d6709)

The effect I wanted to remove was the "shadow" on the right side of this search tip, but I do see now that it also squishes the menu items together. I think I'll remove the rule and add a more specific one to target those items. For sure give a heads-up to Jason. 👍